docs/discover/try-esql.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[try-esql]]
+== Try ES|QL
+
+preview::[]
+
+The Elasticsearch Query Language, {esql}, makes it easier to explore your data without leaving Discover. 
+
+In this tutorial we'll use the {kib} sample web logs in Discover and Lens to explore the data and create visualizations. 
+
+[float]
+[[prerequisite]]
+=== Prerequisite 
+
+To be able to select **Try {esql}** from the Data views menu the `discover:enableESQL` setting must be enabled from **Stack Management > Advanced Settings**. It is enabled by default. 
+
+[float]
+[[tutorial-try-esql]]
+=== Trying ES|QL 
+
+To load the sample data:
+
+. On the home page, click **Try sample data**.
+. Click **Other sample data sets**.
+. On the Sample web logs card, click **Add data**.
+. Open the main menu and select *Discover*.
+. From the Data views menu, select *Try ES|QL*.
```

Comment:
```suggestion
. From the Data views menu, select *Try {esql}*.
```

Reply:
Thanks for catching all these! Sorry about missing the attribute